docs/source/general/freeradius.rst

```diff
@@ -283,6 +283,11 @@ Restart freeradius to load the new configuration:
     # alternatively if you are using systemd
     systemctl restart freeradius
 
+In case of any error, run ``journalctl -xe`` to know the cause. You can also run ``freeradius -X`` to know the 
+detailed cause. One error might be that the django-freeradius server might not be running in the background.
+To set up the django-freeradius server refer `this <https://github.com/YashJipkate/django-freeradius.git>`_
```

Comment:
why you referer django-freeradius to your repo?

Reply:
Oh I am sorry, didn't notice that blunder. Seems like its a bad day.
Thanks for pointing that out @ppabcd !

Reply:
I have corrected it and pushed again. Please review it.
Thanks.